src/components/graph/menus/create-node-menu.tsx

```diff
@@ -394,73 +369,113 @@ const CreateNodeMenu: React.FC<CreateNodeMenuProps> = ({
                     disabled: !isNodePastingAllowed() || isNodeInsertionForbidden(NodeInsertModes.NewBranch),
                 },
-                PASTE_MODIFICATION_NODE_BEFORE: {
+                PASTE_BEFORE: {
                     onRoot: false,
                     action: () => pasteNetworkModificationNode(NodeInsertModes.Before),
-                    id: 'pasteNetworkModificationNodeAbove',
+                    id: 'insertNodeAbove',
                     disabled: !isNodePastingAllowed() || isNodeInsertionForbidden(NodeInsertModes.Before),
                 },
-                PASTE_MODIFICATION_NODE_AFTER: {
+                PASTE_AFTER: {
                     onRoot: true,
                     action: () => pasteNetworkModificationNode(NodeInsertModes.After),
-                    id: 'pasteNetworkModificationNodeBelow',
+                    id: 'insertNodeBelow',
                     disabled: !isNodePastingAllowed() || isNodeInsertionForbidden(NodeInsertModes.After),
                 },
             },
         },
-        UNBUILD_NODE: {
-            onRoot: false,
-            action: () => unbuildNode(),
-            id: 'unbuildNode',
-            disabled: !isNodeUnbuildingAllowed(),
-        },
         REMOVE_NODE: {
             onRoot: false,
             action: () => removeNode(),
             id: 'removeNode',
             disabled: !isNodeRemovingAllowed(),
             sectionEnd: true,
-        },
-        COPY_SUBTREE: {
-            onRoot: false,
-            action: () => copySubtree(),
-            id: 'copyNetworkModificationSubtree',
-            disabled: isAnyNodeBuilding || !isNodeHasChildren(activeNode, treeModel),
-        },
-        CUT_SUBTREE: {
-            onRoot: false,
-            action: () => (isSubtreeAlreadySelectedForCut() ? cancelCutSubtree() : cutSubtree()),
-            id: isSubtreeAlreadySelectedForCut()
-                ? 'cancelCutNetworkModificationSubtree'
-                : 'cutNetworkModificationSubtree',
-            disabled: isAnyNodeBuilding || !isNodeHasChildren(activeNode, treeModel),
-            sectionEnd: true,
-        },
-        PASTE_SUBTREE: {
-            onRoot: true,
-            action: () => pasteSubtree(),
-            id: 'pasteNetworkModificationSubtree',
-            disabled: !isSubtreePastingAllowed() || !isSubtreeContentPasteable(),
-        },
-        REMOVE_SUBTREE: {
-            onRoot: false,
-            action: () => removeSubtree(),
-            id: 'removeNetworkModificationSubtree',
-            disabled: !isSubtreeRemovingAllowed(),
+            withDivider: isSecurityModificationNode(activeNode),
         },
         RESTORE_NODES: {
             onRoot: true,
             hidden: isSecurityModificationNode(activeNode),
             action: () => restoreNodes(),
             id: 'restoreNodes',
             disabled: !isNodeRestorationAllowed(),
+            withDivider: !isSecurityModificationNode(activeNode),
```

Comment:
```suggestion
            withDivider: true,
```

if (!isSecurityModificationNode(activeNode)) is false, this is hidden